flannel/flanneld-run.sh

```diff
@@ -4,10 +4,12 @@
 echo "[Service]" > /etc/systemd/system/docker.service.d/$NAME.conf
 echo "EnvironmentFile=-/run/$NAME/docker" >> /etc/systemd/system/docker.service.d/$NAME.conf
 
+source /etc/sysconfig/flanneld
+
 # Ensure this file doesn't already exist.
 rm -f run/flannel/subnet.env
 
-NOTIFY_SOCKET=/dev/null /usr/bin/flanneld &
+NOTIFY_SOCKET=/dev/null /usr/bin/flanneld -etcd-endpoints=${FLANNEL_ETCD_ENDPOINTS} -etcd-prefix=${FLANNEL_ETCD_PREFIX} -etcd-cafile=${FLANNEL_ETCD_CAFILE} -etcd-certfile=${FLANNEL_ETCD_CERTFILE} -etcd-keyfile=${FLANNEL_ETCD_KEYFILE} $FLANNEL_OPTIONS &
```

Comment:
I just tested it w/o providing values for the ca, cert, keyfile, and was able to use etcd insecurely.